frontend/src/app/modules/interventions/interventions.service.ts

```diff
@@ -48,6 +48,12 @@ export class InterventionsService {
       .pipe(map(toIntervention), this.snackBar.failurePipe());
   }
 
+  submitComment(comment: string, id: string): Observable<any> {
+    return this.http.post(`${this.interventionsUrl}/${id}/comments`, {
+      comment,
+    });
```

Comment:
OK. Done. I've added also success notification.
I've changed `this.openSnackBar` to `this.snackBar.open`. The former was not displaying message in case of error.

Reply:
Resolving this, but please check further discussion here: https://github.com/lightning-team/Ekostraz/pull/78#discussion_r383023608